src/components/app/details/cicdHistory/Sidebar.tsx

```diff
@@ -157,6 +159,16 @@ const HistorySummaryCard = React.memo(
             return `${generatePath(path, _params)}/${currentTab}`
         }
 
+        useEffect( () => {
+            scrollToElement()
+        }, [])
+
+        const activetriggerId = params['triggerId']
+        const scrollToElement = () => {
+            if(targetCardRef && targetCardRef.current){
```

Comment:
use targetCardRef?.current instead & format the file

Reply:
done